src/index.html

```diff
@@ -5,9 +5,221 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" href="images/favicon.png">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,400&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <div class="wrap">
+      <header class="header">
+        <div class="header__top">
+          <div class="header__phone">
+            <a href="tel:+12345555555" class="header__link-number">+1 234 555-55-55</a>
+          </div>
+          <div class="logo header__logo"></div>
+          <div class="header__action">
+            <a href="tel:+12345555555" class="header__link">Call to order</a>
+          </div>
+        </div>
+        <div class="header__title">
+          <h1 class="header__name">Miami<br>Condo Kings</h1>
+        </div>
+        <nav class="nav header__nav">
+          <ul class="nav__list">
+            <li class="nav__item nav__item-active">Home</li>
+            <li class="nav__item"><a href="#about" class="nav__link">About Us</a></li>
+            <li class="nav__item"><a href="#gallery" class="nav__link">Gallery</a></li>
+            <li class="nav__item"><a href="#consultation" class="nav__link">Consultation</a></li>
+            <li class="nav__item"><a href="#testimonials" class="nav__link">Testimonials</a></li>
+            <li class="nav__item"><a href="#contact" class="nav__link">Contacts</a></li>
+          </ul>
+        </nav>
+      </header>
+    </div>
+    <main class="container">
+      <section class="section" id="about">
+        <div class="info">
+          <h2 class="info__heading">Miami best<br>Real estate <a href="https://goo.gl/maps/DHGXgDuzmszWuisH8" class="link">here</a></h2>
+          <div class="info__text">
+            <p class="info__text-colored">Over 50 years experience in love.</p>
+            <p>Mazel and VA Tsukkerman in his "Analysis of musical works."<br>
+              Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.<br><br>
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. <br>
+              Working directly with developers - so have more useful information about the profitable deals.<br><br>
+              <strong> We will never be late! <br>
+              Guaranteed secure transaction - 100%</strong></p>
```

Comment:
кто так пишет теги?
```suggestion
              <strong>
                We will never be late! <br>
                Guaranteed secure transaction - 100%
              </strong>
            </p>
```

проверь весь код чтобы везде были корректные переносы, открывающие и закрывающие теги